### ISSUE TITLE
Unable to remove server when server does not exist

### ISSUE THREAD

**Issue by jonaharagon** [DagAgren/toot#507]
Deleting a server from the "Other Servers" list doesn't seem to work when:

1. You are logged in to that server
2. The server is not available

I assume it's trying to log out, but if logging out isn't available then it's impossible to remove.

**Comment by ChlorideCull** [comment]
Yeah, I've been stuck with snouts.online since that went down. It should probably ask you if you want to remove it without logging out if the server either doesn't resolve, or responds in an unexpected way.
